bindings/bindingtester/bindingtester.py

```diff
@@ -49,6 +47,8 @@
 import fdb
 import fdb.tuple
 
+sys.path[:0] = [os.path.join(os.path.dirname(__file__), '..')]
```

Comment:
I'm not certain about moving this, since I think that might mess up some of the imports. We might need to address this a different way if so

Reply:
Yes, you're right. I made a mistake.
We might have to find a better way of doing these imports though, as said in [this](https://stackoverflow.com/questions/1893598/pythonpath-vs-sys-path) stack overflow post with setuptools. 
I'll revert it.

Reply:
I made an issue for it #8708